nnpdf_data/nnpdf_data/theory_cards/391.yaml

```diff
@@ -0,0 +1,48 @@
+ID: 391
+PTO: 1
+FNS: FONLL-B
+DAMP: 0
```

Comment:
I think pch needs damping (here and below) cc @RoyStegeman

Reply:
Yes, you're right. `pineko` takes as input the powers to the damping function so probably you want to add 
```
DAMPPOWERc: 2
DAMPPOWERb: 2
```

Reply:
But only at NLO ?